tests/test_stats/rasterize/rasterize_pipeline.json

```diff
@@ -2,123 +2,193 @@
-        1,
-        "int32",
-        0,
-        31944,
-        0.0,
-        327.0,
-        11.40449,
-        0.0,
-        30027,
-        11,
-        693,
-        0,
-        124,
-        386,
-        567,
-        136,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0
-      ]
+      "CROP_ID": {
+        "array_shape": [
+          121,
+          264,
+          1
+        ],
+        "dtype": "int32",
+        "random_values": [
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0
+        ],
+        "values": [
+          {
+            "count": 29638,
+            "value": 0
+          },
+          {
+            "count": 817,
+            "value": 1
+          },
+          {
+            "count": 400,
+            "value": 4
+          },
+          {
+            "count": 953,
+            "value": 5
+          },
+          {
+            "count": 136,
+            "value": 14
+          }
+        ]
+      },
+      "POLYGON_ID": {
+        "array_shape": [
+          121,
+          264,
+          1
+        ],
+        "basic_stats": {
+          "max": 327,
+          "mean": 11.40449,
+          "median": 0.0,
+          "min": 0
+        },
+        "counts": {
+          "infinite": 0,
+          "nan": 0
+        },
+        "dtype": "int32",
+        "histogram": {
+          "counts": [
+            30027,
+            11,
+            693,
+            0,
+            124,
+            386,
+            567,
+            136
+          ],
+          "edges": [
+            0.0,
+            40.875,
+            81.75,
+            122.625,
+            163.5,
+            204.375,
+            245.25,
+            286.125,
+            327.0
+          ]
+        },
+        "random_values": [
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0
+        ]
+      }
     }
   },
   "eopatch-id-1-col-0-row-1": {
     "bbox": "BBox(((729480.0, 4391145.0), (732120.0, 4392355.0)), crs=CRS('32638'))",
     "mask_timeless": {
-      "CROP_ID": [
-        121,
-        264,
-        1,
-        "int32",
-        0,
-        31944,
-        0.0,
-        14.0,
-        1.26017,
-        0.0,
-        27465,
-        0,
-        2648,
-        0,
-        0,
-        0,
-        0,
-        1831,
-        0.0,
-        0.0,
-        14.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0
-      ],
-      "POLYGON_ID": [
-        121,
-        264,
-        1,
-        "int32",
-        0,
-        31944,
-        0.0,
-        334.0,
-        42.86069,
-        0.0,
-        26047,
-        37,
-        1013,
-        710,
-        414,
-        537,
-        1355,
-        1831,
-        0.0,
-        0.0,
-        309.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0
-      ]
+      "CROP_ID": {
```

Comment:
Again, I'll trust the numbers are correct.

Reply:
Should be ok. After all, before making this change I ran all pipelines and made sure they work with the old stats.